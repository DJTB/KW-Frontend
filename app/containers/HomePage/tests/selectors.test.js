import selectHomePageDomain, { makeSelectHomePage } from '../selectors';

describe('HomePage selectors', () => {
  const state = { homepage: 'winner!' };
  const HomePageDomain = selectHomePageDomain()(state);
  const selectHomePage = makeSelectHomePage();

  it('selecting homepage domain state should match snapshot', () => {
    expect(HomePageDomain).toMatchSnapshot();
  });

  it('selecting homepage substate should match snapshot', () => {
    expect(selectHomePage(state)).toMatchSnapshot();
  });
});
