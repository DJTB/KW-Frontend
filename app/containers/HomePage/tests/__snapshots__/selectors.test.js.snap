// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage selectors selecting homepage domain state should match snapshot 1`] = `"winner!"`;

exports[`HomePage selectors selecting homepage substate should match snapshot 1`] = `"winner!"`;
